attributes/default.rb

```diff
@@ -81,13 +82,13 @@
 default['hopslog']['private_ips']         = ['10.0.2.15']
 default['hopslog']['public_ips']          = ['10.0.2.15']
 
-# Kibana Opendistro Security plugin
-default['kibana']['opendistro_security']['url']                                   = "#{node['download_url']}/opendistro_security_kibana_plugin-#{node['elastic']['opendistro']['version']}.zip"
-default['kibana']['opendistro_security']['https']['enabled']                      = "true"
-default['kibana']['opendistro_security']['multitenancy']['global']['enabled']     = "false"
-default['kibana']['opendistro_security']['multitenancy']['private']['enabled']    = "true"
-default['kibana']['opendistro_security']['cookie']['ttl']                         = node['elastic']['opendistro_security']['jwt']['exp_ms'].to_i
+# Kibana Opensearch Security plugin
+default['kibana']['opensearch_security']['url']                                   = "#{node['download_url']}/opensearch_security_kibana_plugin-#{node['elastic']['opensearch']['version']}.zip"
```

Comment:
why do you need the plugin isn't it installed by default as part of opensearch_dashboard?